src/index.ts

```diff
@@ -216,7 +216,8 @@ class NestedContextMenu extends BaseContextMenu {
     let { normalizedX, normalizedY } = normalizePozition(
```

Comment:
Since `this.options.normalizePosition` is a flag used to disable this functionality, we should check it's value before calling the `normalziePozition` function.

```js
let { normalizedX, normalizedY } = this.options.normalizePosition ? normalizePozition( { x: parentX, y: parentY }, contextMenu, this.options.scope, this.options.normalizePosition) : {normalizedX: parentX, normalizedY: parentY}
```

Reply:
I tried to refactor this, because this functionality is needed twice. Hope it can be understood easier.